utils/Makefile

```diff
@@ -0,0 +1,5 @@
+print-document-plan:
+	clojure -M:dp print-plan $(dp)
+
+export-all-document-plans:
```

Comment:
maybe we should also include this rule in root Makefile? for example, rule named "export-document-plans" that exports them to "api/resources/document-plans" by default.